manager/state/raft/raft_test.go

```diff
@@ -286,6 +290,7 @@ func TestRaftLeaderLeave(t *testing.T) {
 	t.Parallel()
 
 	nodes, clockSource := raftutils.NewRaftCluster(t, tc)
+	defer raftutils.TeardownCluster(t, nodes)
```

Comment:
`TeardownCluster` is called at the end of the function on `newCluster`, which contains two out of the three nodes.